Gruntfile.coffee

```diff
@@ -88,9 +88,9 @@ module.exports = (grunt) ->
   tasks =
     # Building block tasks
     build: ['clean', 'webpack:main', 'copy:main', 'copy:dev', 'copy:pkg']
-    test: ['build', 'mochaTest']
+    test: ['mochaTest']
     # Deploy tasks
-    dist: ['build', 'mochaTest', 'coffee:dist'] # Dist
+    dist: ['build', 'mochaTest', 'coffee:dist', 'copy:deploy'] # Dist
```

Comment:
Unlike checkout-ui, there is no minification step. Should we add it here or it's being minified somewhere else?
```suggestion
    dist: ['build', 'min', 'mochaTest', 'coffee:dist', 'copy:deploy'] # Dist
```

Reply:
it already is being minified, I believe it is the `webpack` step inside `build` that is doing this